New Tracing Quickstart.md

```diff
@@ -0,0 +1,148 @@
+# New Tracing Quickstart
```

Comment:
This shouldn't be at the top level.

Reply:
@Jimbo4350, done

Reply:
The idea was to temporarily increase visibility of the `Quickstart` for the new tracing system -- as we would like people to start using it.

But I guess it could live in `doc/` as well.